content/en/tracing/advanced/runtime_metrics.md

```diff
@@ -49,9 +49,21 @@ By default, JVM metrics from your application are sent to the Datadog Agent over
 {{% /tab %}}
 {{% tab "Python" %}}
 
-Coming Soon. Reach out to [the Datadog support team][1] to be part of the beta.
+<div class="alert alert-info">
+This feature is currently in <strong>BETA</strong>.
+Reach out to <a href="/help">the Datadog support team</a> to be part of the beta.
+</div>
 
-[1]: /help
+Runtime metrics collection can be enabled with one environment parameter when running with `ddtrace-run`:
```

Comment:
other languages call out using dogstatsd and port 8125. Are we not using that here?